src/tests/java/picard/illumina/CheckIlluminaDirectoryTest.java

```diff
@@ -51,28 +51,30 @@ private void setUp() throws Exception {
         illuminaDir = IOUtil.createTempDir("ift_test", "IlluminaDir");
 
         interopDir = new File(illuminaDir, "InterOp");
-        if (!interopDir.mkdir()) {
+        if (!interopDir.exists() && !interopDir.mkdir()) {
```

Comment:
I don't understand why this is needed still.  If this ```setup``` method is called for this class and this directory is specific for this set of tests, why would it already exist?  Is there another class that creates the same directory?  If so, we should modify the ```IOUtil.createTempDir``` call, even though I think it should create a unique tmp dir?  Basically, we should not need this check *and* be able to do parallel execution of tests.